CONFIGURATION.md

```diff
@@ -0,0 +1,174 @@
+
+For an example `formats` field, see the Example Configuration below.
+
+### Example Configuration
+
+```json
+[
+  {
+    "vocabulary_name": "Neurobagel Phenotypic Variables",
+    "namespace_prefix": "nb",
+    "namespace_url": "http://neurobagel.org/vocab/",
+    "version": "1.0.0",
+    "standardized_variables": [
+      {
+        "name": "Participant ID",
+        "id": "ParticipantID",
+        "data_type": null,
+        "terms_file": null,
+        "formats": null,
+        "required": true,
+        "description": "Describe how to annotate this standardized variable.",
+        "is_multi_column_measure": false,
+        "can_have_multiple_columns": false,
+        "same_as": null
+      },
+      {
+        "name": "Age",
+        "id": "Age",
+        "data_type": "Continuous",
+        "terms_file": null,
+        "formats": [
+          {
+            "id": "FromFloat",
+            "name": "float",
+            "examples": ["31.5"]
+          },
+          {
+            "id": "FromEuro",
+            "name": "euro",
+            "examples": ["24,2", "32,3"]
+          }
+        ],
+        "required": false,
+        "description": "Describe how to annotate this standardized variable.",
+        "is_multi_column_measure": false,
+        "can_have_multiple_columns": false,
+        "same_as": null
+      },
+      {
+        "name": "Assessment tool",
+        "id": "Assessment",
+        "data_type": null,
+        "terms_file": "assessmentsTerms.json",
+        "formats": null,
+        "required": false,
+        "description": "Describe how to annotate this standardized variable.",
+        "is_multi_column_measure": true,
+        "can_have_multiple_columns": true,
+        "same_as": "https://medra.org/terms/1234"
+      },
+      {
+        "name": "Diagnosis",
+        "id": "Diagnosis",
+        "data_type": "Categorical",
+        "terms_file": "diagnosisTerms.json",
+        "formats": null,
+        "required": false,
+        "description": "Describe how to annotate this standardized variable.",
+        "is_multi_column_measure": false,
+        "can_have_multiple_columns": true,
+        "same_as": null
+      }
+    ]
+  }
+]
+```
+
+## Adding Term Files
+
+Term files are only used for categorical variables as they require a predefined set of standardized terms:
+
+1. Create a JSON file in the config directory (e.g., `assessmentTerms.json`)
+2. The file should contain an array of term objects with `id` and `name` fields
+3. Reference the file in the variable's `terms_file` field
+
+### Example Vocabulary File (`assessmentTerms.json`)
+
+```json
+[
+  {
+    "namespace_prefix": "epd",
+    "namespace_url": "https://enigma-parkinson.org/id/",
+    "vocabulary_name": "ENIGMA Parkinson's Assessments",
+    "version": "1.0.0",
+    "terms": [
+      {
+        "name": "Beck's Depression Inventory",
+        "id": "1234",
+        "abbreviation": "BDI",
```

Comment:
do we need a table to describe the optional fields like `abbreviation`, `status`, etc. the same way we do for the fields in config.json above?

Reply:
Hmmm it could be helpful but not sure if this would be the appropriate place for it.